lib/galaxy/webapps/reports/controllers/jobs.py

```diff
@@ -13,6 +13,40 @@
 log = logging.getLogger( __name__ )
 
 
+def sorter(defaultSortID, kwd):
+    """
+    Initialize sorting variables
+    """
+    sort_id = kwd.get('sort_id')
+    order = kwd.get('order')
+
+    # Parse the default value
+    if sort_id == "default":
+        sort_id = defaultSortID
+
+    # Create the sort
+    if order == "asc":
+        _order = sa.asc( sort_id )
+    elif order == "desc":
+        _order = sa.desc( sort_id )
+    else:
+        # In case of default
+        order = "desc"
+        _order = sa.desc( sort_id )
+
+    # Create an arrow icon to put beside the ordered column
+    up_arrow = "&#x2191;"
+    down_arrow = "&#x2193;"
+    arrow = " "
+
+    if order == "asc":
+        arrow += down_arrow
+    else:
+        arrow += up_arrow
+
+    return [sort_id, order, arrow, _order]
```

Comment:
These look like four values with different ... semantic meaning... so a list is probably inappropriate here. A tuple would be more appropriate here ``(sort_id, order, arrow, _order)`` (with parens instead of brackets). Given the way you are using sorter - I don't think any of the call sites would need to change.

I think you should make the above change for this PR. In future development, it might be worth actually looking at using [named tuples](https://docs.python.org/2/library/collections.html#collections.namedtuple) for something l like this.

Create a spec with something like:
```
SortSpec = namedtuple('SortSpec', ['sort_id', 'order', 'arrow', '_order'])
```

Then in sorter you could return ``SortSpec(sort_id, order, arrow, _order)``.

That doesn't really help anything for sorter - but then at the call sites you could do something like

```
        specs = sorter( 'date', kwd )
        sort_id = specs.sort_id
        order = specs.order
        arrow = specs.arrow
        _order = specs._order
```

instead of 

```
        specs = sorter( 'date', kwd )
        sort_id = specs[0]
        order = specs[1]
        arrow = specs[2]
        _order = specs[3]
```

Which I would argue is less brittle and more extensible.